patches/minecraft/net/minecraft/commands/CommandSourceStack.java.patch

```diff
@@ -8,4 +8,4 @@
 +public class CommandSourceStack implements SharedSuggestionProvider, net.minecraftforge.common.extensions.IForgeCommandSourceStack {
     public static final SimpleCommandExceptionType f_81286_ = new SimpleCommandExceptionType(Component.m_237115_("permissions.requires.player"));
     public static final SimpleCommandExceptionType f_81287_ = new SimpleCommandExceptionType(Component.m_237115_("permissions.requires.entity"));
-    private final CommandSource f_81288_;
+    public final CommandSource f_81288_;
```

Comment:
Bad patch change.

Reply:
why does it do this?

Reply:
Either you made the change, or a PR added the AT line which you didn't catch when setting up and regenning the patches. In any case, when it happens, just don't commit the file.

Reply:
I didn't change anything, so how would someones change to AT have made a new patch in my end?

Reply:
whatever happened, I removed it

Reply:
Probably something that happened during rebase.

Reply:
I have been notified by Ater and XFact that the patch is because of sciwhiz adding an AT for the field but forgetting to regen the patches afterwards.